client/js/components/content-slider.js

```diff
@@ -38,7 +38,7 @@ const contentSlider = (el, options) => {
   function setModifiers(cssBlock) {
     return settings.modifiers.reduce((acc, curr) => {
       return `${acc} ${settings.cssPrefix}${cssBlock}--${curr}`;
-    }, '');
+    }, '').trim();
```

Comment:
It's the result of a reduction: https://github.com/wellcometrust/wellcomecollection.org/pull/911#discussion_r118221207